crates/ra_hir/src/expr.rs

```diff
@@ -190,6 +190,10 @@ pub enum Expr {
         args: Vec<ExprId>,
         generic_args: Option<GenericArgs>,
     },
+    MacroCall {
+        macro_name:Name,
+        path:Option<Path>
+    },
```

Comment:
I think by the time we get an `hir::Expr`, all macros should be expanded. So, instead of adding an `MacroCall` here, we should eagarly expand it when lowering